test/es7/asyncawait-syntax.js

```diff
@@ -1,10 +1,14 @@
 //-------------------------------------------------------------------------------------------------------
 // Copyright (C) Microsoft. All rights reserved.
+// Copyright (c) ChakraCore Project Contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
 // ES6 Async Await tests -- verifies syntax of async/await
 
+// @ts-check
+/// <reference path="..\\UnitTestFramework\\UnitTestFramework.js" />
+
```

Comment:
Please could you remove this I know it's useful within vscode but for now it's not standard in our source tree

Reply:
Sure, but I'm afraid I've already added [some](https://github.com/search?q=repo%3Achakra-core%2FChakraCore%20%2F%2F%20%40ts-check&type=code) into CCs code-base 😅